sdk/tests/deqp/framework/common/tcuTestCase.js

```diff
@@ -271,7 +272,6 @@ goog.scope(function() {
 
         // If no more children, get the next brother
         if (test == null && this.parentTest != null) {
-            this.currentTestNdx = 0;
```

Comment:
Out of curiosity what bug did this fix? I noticed this in the original code and it seemed both confusing and suspicious, but I decided not to touch it to avoid potentially breaking something.